OrbitGl/TimeGraph.cpp

```diff
@@ -196,16 +196,18 @@ void TimeGraph::ZoomTime(float a_ZoomValue, double a_MouseRatio) {
   SetMinMax(minTimeUs, maxTimeUs);
 }
 
-void TimeGraph::VerticalZoom(float zoom_value, double mouse_relative_position) {
-  static double increment_ratio = 0.1;
+void TimeGraph::VerticalZoom(float zoom_value, float mouse_relative_position) {
+  static const float increment_ratio = 0.1f;
```

Comment:
Nit: That looks surprising. No need to make it static if it's const. So I would suggest to remove the static and maybe replace the const by constexpr.